src/Our.Umbraco.Ditto/ComponentModel/TypeConverters/MultiNodeTreePickerConverter.cs

```diff
@@ -58,7 +60,9 @@ public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo c
                 var multiNodeTreePicker = Enumerable.Empty<T>();
 
                 int n;
-                var nodeIds = s
+                var nodeIds = XmlHelper.CouldItBeXml(s)
+                    ? uQuery.GetXmlIds(s)
```

Comment:
For ref, the code for `uQuery.GetXmlIds()` is here:
https://github.com/umbraco/Umbraco-CMS/blob/dev-v7/src/Umbraco.Web/umbraco.presentation/umbraco/uQuery/uQuery.cs#L143